WordPress/Classes/ViewRelated/Stats/Insights/TabbedTotalsCell.swift

```diff
@@ -101,7 +113,6 @@ private extension TabbedTotalsCell {
     func applyStyles() {
         Style.configureCell(self)
         Style.configureLabelAsTotalCount(totalCountLabel)
-        Style.configureViewAsSeperator(topSeparatorLine)
         Style.configureViewAsSeperator(bottomSeparatorLine)
```

Comment:
Not a new method, but perhaps we could correct the spelling (i.e., sep**a**rator)?

Reply:
Oh man. I'm _always_ misspelling separator!